scripts/g.bands/landsat.json

```diff
@@ -0,0 +1,41 @@
+{
+    "Landsat7": {
+        "description": "The Landsat 7 bands",
+        "shortcut": "Ls7",
```

Comment:
Do we want "Ls7" or L7? (say, is "Ls7" a common abbreviation?)

Reply:
I haven't seen Ls7 or Ls8 anywhere; common use is L7, L8 and such. It's more comfortable to write as well, hence I'd prefer the later.

Reply:
`Ls7` shortcut changed to `L7` in 3a4dae5e93.

```
> g.bands band=L7
L7_1
L7_2
L7_3
L7_4
L7_5
L7_6
L7_7
L7_8
```